terraform/create_backend_vars.sh

```diff
@@ -11,9 +11,12 @@ fi
 ACCESS_KEY=$(echo "${S3_CREDENTIALS}" | jq -r '.access_key_id')
 SECRET_KEY=$(echo "${S3_CREDENTIALS}" | jq -r '.secret_access_key')
 REGION=$(echo "${S3_CREDENTIALS}" | jq -r '.region')
+BUCKET=$(echo "${S3_CREDENTIALS}" | jq -r '.bucket')
 
 {
   echo "access_key = \"$ACCESS_KEY\""
   echo "secret_key = \"$SECRET_KEY\""
   echo "region = \"$REGION\""
+  echo "bucket = \"$BUCKET\""
```

Comment:
We weren't able to use variables inside of the terraform file at the backend call site, so we are adding it here and removing it from the .tf file.